src/execution/execute.js

```diff
@@ -851,7 +851,7 @@ function completeValue(
       throw new Error(
         `Cannot return null for non-nullable field ${info.parentType.name}.${
           info.fieldName
-        }.`,
+        }. Value ${result} cannot be converted to ${returnType.ofType.name}`,
```

Comment:
> The pipeline failed but I'm not sure why. TBH it's the first time I do a PR for a flow project (I'm usually using Typescript).

Because you can have types like `[String]!` and `returnType.ofType` would be equal to `[String]`. You can fix it like that:
```suggestion
        }. Value ${result} cannot be converted to ${inspect(returnType)}`,
```

Reply:
thanks!